iroha/src/lib.rs

```diff
@@ -30,7 +30,7 @@ use crate::{
     peer::Peer,
     prelude::*,
     queue::Queue,
-    sumeragi::{Message, Sumeragi},
+    sumeragi::{message::Message as SumeragiMessage, Sumeragi},
```

Comment:
It's still the only message here) Please do not alias it.

Reply:
Sure, though still, it seem more intuitive for me with `SumeragiMessage` :sweat_smile: